packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Create an API token
+excerpt: How to create a token for the Okta API
+layout: Guides
+---
+
+This page explains what an API token is, why you need one, and how to create one.
+
+**Learning outcomes**
```

Comment:
I see in your nutrition facts you haven't included "Sample code" with an entry of n/a because there isn't any. Christine and I agreed that we would do this, just for maximum consistency, and so the reader gets used to what content is commonly found there. Would you prefer to just not include sections that have no content? I'm happy to have a chat about this one; I'm a bit on the fence about it.